core-test/src/main/java/org/openstack4j/api/storage/VolumeTests.java

```diff
@@ -63,8 +63,6 @@ public void getVolumeV1() throws Exception {
         assertTrue(getRequest.getPath().matches("/v[12]/\\p{XDigit}*/volumes/8a9287b7-4f4d-4213-8d75-63470f19f27c"));
         
         assertEquals(volume.getId(), "8a9287b7-4f4d-4213-8d75-63470f19f27c");
-        assertEquals(volume.getName(), "vol-test");
```

Comment:
Done. Fixed both classes, added `display_name` and `display_description` to maintain backwards compatibility with deprecated cinder v1 API.